commit 578739b21caecd45a1c84d6ea4837e8a9ecee516
Author: Krzysztof Opasiak <kopasiak90@gmail.com>
Date:   Thu Jan 19 18:55:29 2017 +0100

    usb: gadget: f_hid: Use spinlock instead of mutex
    
    commit 33e4c1a9987a1fc3b42c3b534100b5b006d55c61 upstream.
    
    As IN request has to be allocated in set_alt() and released in
    disable() we cannot use mutex to protect it as we cannot sleep
    in those funcitons. Let's replace this mutex with a spinlock.
    
    Tested-by: David Lechner <david@lechnology.com>
    Signed-off-by: Krzysztof Opasiak <k.opasiak@samsung.com>
    Signed-off-by: Felipe Balbi <felipe.balbi@linux.intel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1315_578739b21caecd45a1c84d6ea4837e8a9ecee516.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1315_578739b21caecd45a1c84d6ea4837e8a9ecee516.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
